### ISSUE TITLE
no instances to display error

### ISSUE THREAD

**Issue by bellakate** [SUYEgit/Surgery-Robot-Detection-Segmentation#5]
hello sir.. i trained on multiple classes and the training was successful. when i try to use the model to detect on a single image, it shows no instances to display error. Can you please tell me what causes the error?
any help will be greatly appreciated!

**Comment by SUYEgit** [comment]
Hi, thank you for using the codes. 

No instance means that the model you trained can't detect any instance that belongs to classes you defined. I recommend:
1. Check the training loss to see if your model has converged.
2. If the loss has already come to a flat area, maybe you can check if your model has already overfit the training set by detecting images in training set.
